src/views/Batches/batchComponents/BatchForm.jsx

```diff
@@ -489,7 +495,19 @@ const BatchForm = (props) => {
                       placeholder=""
                     />
 
-                </div>
+                  </div>
+                  <div className="col-md-6 col-sm-12 mt-2">
+                       <Input
+                        control="lookup"
+                        icon="down"
+                        name="Mode_of_Payment"
```

Comment:
Some questions: 
1. Is this field getting saved also?
2. We also need to show the saved value in the batch details page, below `Per-student Fees`
<img width="929" alt="image" src="https://user-images.githubusercontent.com/12053186/230868264-6f6ea3b7-ab3d-4930-8199-7044338d644e.png">